build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+DESCRIPTION="HyprIoT Cluster Lab".
+DEPENDENCIES="avahi-utils, vlan, dnsmasq"
+BUILD_DIR="./"
+
+VERSION="$(cat VERSION)"
+PACKAGE_VERSION=${VERSION}-${1:-"1"}
+
+REPO_NAME="$(basename "$(git rev-parse --show-toplevel)")"
+PROJECT_NAME=hypriot-${REPO_NAME}
+PACKAGE_NAME="${PROJECT_NAME}_${PACKAGE_VERSION}"
+TIMESTAMP="$(date +"%Y-%m-%d_%H%M")"
+COMMIT="$(git rev-parse --short HEAD)"
+
+rm -f ${BUILD_DIR}/${PACKAGE_NAME}.deb
+
+mkdir -p ${BUILD_DIR}/${PACKAGE_NAME}
+cp -r package/* ${BUILD_DIR}/${PACKAGE_NAME}/
+
+cp cluster-lab_images.list ${BUILD_DIR}/${PACKAGE_NAME}/etc/cluster-lab/
+cp imageloader.sh ${BUILD_DIR}/${PACKAGE_NAME}/usr/local/bin/imageloader
+chmod +x ${BUILD_DIR}/${PACKAGE_NAME}/usr/local/bin/imageloader
+
+sed -i'' "s/<VERSION>/${PACKAGE_VERSION}/g" ${BUILD_DIR}/${PACKAGE_NAME}/DEBIAN/control
+sed -i'' "s/<NAME>/${PROJECT_NAME}/g" ${BUILD_DIR}/${PACKAGE_NAME}/DEBIAN/control
+sed -i'' "s/<SIZE>/60/g" ${BUILD_DIR}/${PACKAGE_NAME}/DEBIAN/control
```

Comment:
This seems hardcoded. All sed commands could be combined into one so you don't have to repeat `${BUILD_DIR}/${PACKAGE_NAME}/DEBIAN/control`. e.g. 
```bash
sed -i'' "s/<VERSION>/${PACKAGE_VERSION}/g;s/<NAME>/${PROJECT_NAME}/g;s/<SIZE>/60/g;s/<DESCRIPTION>/${DESCRIPTION}/g;s/<DEPENDS>/${DEPENDENCIES}/g" ${BUILD_DIR}/${PACKAGE_NAME}/DEBIAN/control
```
with maybe better formatting